[TASK] Introduce cards on startpage (#4947)
TYPO3-Documentation/TYPO3CMS-Reference-CoreApi: Documentation/Index.rst

..  include:: /Includes.rst.txt

..  _start:
..  _api-overview:

===============
TYPO3 Explained
===============

..  card-grid::
    :columns: 1
    :columns-md: 2
    :gap: 4
    :class: pb-4
    :card-height: 100

    ..  card:: :ref:`API A-Z <api>`

>         This chapter describes the most important elements of the API.

Reviewer: For me "it describes API elements" is too short. I do not know what the API elements are. So maybe provide some examples or add a few further words to it. I have no idea since I am not used to apply API's in my daily life.

Author: Well it is "somehow everything" not sure how to explain this in short

Reviewer 2: How about: "This chapter describes the core functionality of TYPO3 including API and framework configuration"?

Author: Sounds good to me

>         This chapter describes the most important elements of the API.

Author: ```suggestion
        This chapter describes the core functionality of TYPO3 including API and framework configuration
```

>         This chapter describes the core functionality of TYPO3 including API and framework configuration

Reviewer: ```suggestion
        This chapter describes the core functionality of TYPO3 including API and framework configuration.
```

    ..  card:: :ref:`Administrations <administration>`

        Including :ref:`Installations <installation>`, :ref:`Upgrades and updates <upgrading>`,
        :ref:`Troubleshooting for administrators <troubleshooting-index>`, and
        :ref:`Backend User Management <user-management>`

    ..  card:: :ref:`Configuration <configuration>`

        Including :ref:`typo3ConfVars`, :ref:`typoscript`, the :ref:`config-module`,
        and :ref:`feature-toggles`

    ..  card:: :ref:`Coding guidelines <cgl>`

>         This chapter contains description of the formal requirements or standards

Reviewer: ```suggestion
        This chapter contains a description of the formal requirements or standards
```

        regarding coding that you should adhere to when you develop TYPO3
        extensions or Core parts.

    ..  card:: :ref:`Extension development <extension-development>`

        Learn how to write site packages and other custom extension with or
        without Extbase.

    ..  card:: :ref:`Security guidelines <security>`

        This chapter describes some typical risks and advises how to protect a
        TYPO3 site in order to ensure it is and stays secure and stable.

>     ..  card:: :ref:`Automated testing <security>`

Reviewer: ```suggestion
    ..  card:: :ref:`Automated testing <testing>`
```
